src/plugins/cache/README.md

```diff
@@ -0,0 +1,54 @@
+- infos = Information about the cache plugin is in keys below
+- infos/author = Mihael Pranjic <mpranj@limun.org>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides =
+- infos/recommends =
+- infos/placements = pregetcache postgetcache
+- infos/status = maintained unittest global preview concept
+- infos/metadata =
+- infos/description = caches keysets from previous `kdbGet()` calls
+
+## Introduction
+
+This caching plugin stores keysets from previous `kdbGet()` calls
+to improve performance when reading configuration files.
+
+## Usage
+
+You can use `scripts/copy-cache`
```

Comment:
Please update.

Reply:
Yep. Sorry, but it was not yet ready for review. Some problems which I did not encounter came up on the build servers.

Reply:
I only looked over it now so that we hopefully can immediately merge it once the build server says okay. The obvious TODOs, like the code to be removed, I did not mark, though.